odbc/src/api/functions.rs

```diff
@@ -660,6 +698,17 @@ pub unsafe extern "C" fn SQLColumns(
                 column,
             );
             let mongo_statement = odbc_unwrap!(mongo_statement, mongo_handle);
+            /*
```

Comment:
Perhaps this commented code was left in by accident?

Reply:
Yes :(